### DISCUSSION TITLE
General problem

### DISCUSSION THREAD

**Discussion by suloman001** [giswqs/geog-312#159] [Q&A] [answered]
I am Muhammad Sulaiman, a student of BS 5th semester in GIS/RS. I am very confused because there are so many applications of GIS. As a student of GIS and RS, what kind of skills should I learn first after clearing the basics of GIS and remote sensing? Which skills have the most scope nowadays? and what course I learn and what coding language I learn ?

**Comment by giswqs** [comment, marked as answer]
Learn this course.
